### ISSUE TITLE
WzA - Allegro Odbiór w Punkcie Pocztex - nieistniejący punkt?

### ISSUE THREAD

**Issue by jzby-dev** [allegro/allegro-api#8791]
Przy próbie nadania przesyłki do Punktu Pocztex dostaję informację o niewłaściwym punkcie odbioru, pointId to 414491, jak mogę zweryfikować id tego punktu? Jest gdzieś dostępna lista tych punktów?

**Comment by MaciejFrackowiak** [comment]
Identyfikatory punktów dostępne są w API każdego przewoźnika.

**Comment by jzby-dev** [comment]
Miałem oczywiście na myśli API Allegro, ponieważ żeby dostać się do API operatorów muszę się w jakiś sposób tam rejestrować. Rozumiem że przez API Allegro nie ma takiej możliwości?

**Comment by MaciejFrackowiak** [comment]
Nie, w API Allegro nie zwracamy identyfikatorów punktów przewoźników. W tym przypadku musisz skorzystać z [zasobów Poczty Polskiej](https://www.poczta-polska.pl/webservices/).

**Comment by jzby-dev** [comment]
Ok, dzięki.

**Comment by jzby-dev** [comment]
Listy punktów są dostępne na stronie poczty: https://odbiorwpunkcie.poczta-polska.pl/integracja-sklepu/ (pod nagłówkiem OPROGRAMOWANIE WŁASNE są pliki csv, xml i coś jeszcze). Co jeśli punkty do których chcę nadać paczki (w tej chwili dwa: 414491, 414484) są w tych plikach a API Allegro je odrzuca?

**Comment by MaciejFrackowiak** [comment]
To niestety jest efekt problemów technicznych Poczty Polskiej - otrzymaliśmy potwierdzenie, że PP pracuje już nad wyeliminowaniem tego problemu.
